docs/en/05_Releases_and_changelogs/cwp_recipe_basic_1.9.3.md

```diff
@@ -7,8 +7,27 @@ This CWP release ensures compatiblity with PHP 7.2 and newer
 To upgrade, adjust your projects composer.json to reflect the new constraints:
 
 ```json
+    "require": {
         "cwp/cwp-recipe-basic": "^1.9.3@stable",
-        "cwp/cwp-recipe-blog": "^1.9.3@stable"
+        "cwp/cwp-recipe-blog": "^1.9.3@stable",
+        [...]
+    },
+```
+
+## Multi-factor Authentication (MFA)
+
+In order to comply with CWP agreements in security auditing releases, this version has been released with a minor issue preventing MFA from operating in conjunction with the subsites module. Releases for both Framework and Subsites have been made since in order to rectify this, however as these were not included in the security audit for the 1.9.3 release agencies are encouraged to evaluate the decision to install these, and the impacts for them in deviating from the security audited CWP release.
```

Comment:
```suggestion
This version has been released with a minor issue preventing MFA from operating in conjunction with the subsites module. Releases for both Framework and Subsites have been made since in order to rectify this, however as these were not included in the security audit for the 1.9.3 release agencies are encouraged to evaluate the decision to install these, and the impacts for them in deviating from the security audited CWP release.
```

Reply:
I don't think this sentence is required in the introduction.